client/src/theme/theme.ts

```diff
@@ -4,4 +4,9 @@ export const customTheme = createTheme({
   typography: {
     fontFamily: "var(--font-inter)",
   },
+  palette:{
+    text: {
```

Comment:
Why is the theme for the color for the changed from main to this one?

Reply:
It has been resolved!